test-network-nano-bash/README.md

```diff
@@ -59,6 +59,16 @@ If you have trouble running bash scripts in your environment, you can just as ea
 
 Note the syntax of running the scripts. The peer admin scripts set the admin environment variables and must be run with the `source` command in order that the exported environment variables can be utilized by any subsequent user commands.
 
+## Running each component separately with CAs
+
+These instructions are for running the CAs from terminal sessions.  Open terminal windows for 3 CAs as seen in the following terminal setup. These instructions should be followd before opening the ordering and peer windows described above.
+
+![CA Terminal setup](ca_terminal_setup.png)
+
+- cd to the `test-network-nano-bash` directory in each terminal window
+- Instead of running the  `./generate_artifacts.sh` described above, in the three CA terminals, run `./ordererca.sh`, `./org1ca.sh`, `./org2ca.sh` respectively.
```

Comment:
```suggestion
- Before running `./generate_artifacts.sh`, in the three CA terminals, run `./ordererca.sh`, `./org1ca.sh`, `./org2ca.sh` respectively.
```

Reply:
I have reworded this.  It should make sense now:

`- Before running the  `./generate_artifacts.sh -ca` in the first orderer terminal, run `./ordererca.sh`, `./org1ca.sh`, `./org2ca.sh` in the repsective terminals.`